hathor/manager.py

```diff
@@ -98,7 +99,12 @@ def __init__(self, reactor: IReactorCore, peer_id: Optional[PeerId] = None, netw
         from hathor.metrics import Metrics
         from hathor.p2p.factory import HathorClientFactory, HathorServerFactory
         from hathor.p2p.manager import ConnectionsManager
-        from hathor.transaction.storage.memory_storage import TransactionMemoryStorage
+
+        if not (enable_sync_v1 or enable_sync_v2):
+            raise TypeError(f'{type(self).__name__}() at least one sync version is required')
+
+        if tx_storage is None:
+            raise TypeError(f'{type(self).__name__}() missing 1 required positional argument: \'tx_storage\'')
```

Comment:
We are doing this just to prevent changing the parameter orders if we change to a required parameter in the constructor?

Reply:
Exactly. If we remove the default value, we must change the order.